src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -2262,10 +2359,14 @@ Returns the current routing status of each agent selected by the provided filter
 |                        |                                                                                                |
 | ---------------------- | ---------------------------------------------------------------------------------------------- |
 | **Method URL**         | `https://api.livechatinc.com/v3.4/agent/action/list_routing_statuses`                          |
-| **Required scopes**    | `agents--all:ro`, `agents-bot--all:ro`                                                         |
+| **Required scopes**    | `agents--all:ro` **\*** and `agents-bot--all:ro` **\*\***                                      |
```

Comment:
@bkubiak there was a 'because there is not a param that will allow you to choose which type of agent you want to list' part after scopes listed. I did not put it here. Please confirm if it should not be here :) (same question to the equivalent in RTM)

Reply:
Yes, it definitely shouldn't be here - it is internal info for you :)